src/bitcoind/poller.rs

```diff
@@ -450,7 +450,7 @@ impl ToBeCpfped {
 // CPFP a bunch of transactions, bumping their feerate by at least `target_feerate`.
 // `target_feerate` is expressed in sat/kWU.
 // All the transactions' feerate MUST be below `target_feerate`.
-fn cpfp_package(
+pub fn cpfp_package(
```

Comment:
nit: please make the docstring a doc comment (`///`) if you make this function public